app/javascript/onboarding/Onboarding.jsx

```diff
@@ -70,18 +69,18 @@ export class Onboarding extends Component {
     const { currentSlide } = this.state;
     const { communityConfig } = this.props;
     return (
-      <FocusTrap>
-        <main
-          className="onboarding-body"
-          style={
-            communityConfig.communityBackground && {
-              backgroundImage: `url(${communityConfig.communityBackground})`,
-            }
+      <main
+        className="onboarding-body"
+        style={
```

Comment:
Refactor (blocking): Minor, but if there is no community background, the rendered style attribute value will become `style="false"`.

Reply:
not introduced in this commit but I see what you mean - I'll change it 👍